### PR TITLE
Implement couchbase as backend in rend

### PR DESCRIPTION
**Pull request by geobeau**

### CODE REVIEW

File: handlers/couchbase/couchbase.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+var (
+	errNotFound = "key not found"
+)
+
+type Handler struct {
+	client clientWrapper
+}
+
+type clientWrapper interface {
```
Comment (erebe) [inline handlers/couchbase/couchbase.go:19]: Where is it used ?

File: handlers/couchbase/couchbase.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+type couchbaseClient struct {
+	client *gocb.Bucket
+}
+
+func (c *couchbaseClient) close() error {
+	return c.client.Close()
+}
+
+func (c *couchbaseClient) get(key []byte, data *[]byte) error {
+	_, err := c.client.Get(string(key), data)
```
Comment (erebe) [inline handlers/couchbase/couchbase.go:36]: Those `string(key)` seems very inneficiant to me
Can you put this link in comment if we need it https://github.com/golang/go/issues/25484

Comment (geobeau) [reply handlers/couchbase/couchbase.go:36]: Done

File: handlers/couchbase/couchbase.go (hunk 1 above)
Comment (geobeau) [reply handlers/couchbase/couchbase.go:19]: Above and in test. Basically it's a wrapper in the client so that it's easier to test

**Review by erebe — APPROVED** [review]

**Pull request merged**